src/Workspaces/Core/Portable/Workspace/Workspace.cs

```diff
@@ -737,6 +751,23 @@ await Task.Factory.StartNew(() =>
         }, cancellationToken, TaskCreationOptions.None, _taskSchedulerProvider.CurrentContextScheduler).ConfigureAwait(false);
     }
 
+    private static async ValueTask ProcessBackgroundWorkQueueAsync(ImmutableSegmentedList<Func<Task>> list, CancellationToken cancellationToken)
+    {
+        foreach (var item in list)
+        {
+            cancellationToken.ThrowIfCancellationRequested();
+
+            try
+            {
+                await item().ConfigureAwait(false);
```

Comment:
curious if there's any desire/need to pass the ct along through this callback.  did you notice any cases where the callback would have benefitted?

Reply:
definitely should pass that through, will do in next commit

Reply:
No longer applicable as the support for async events was removed